tekton/ci/jobs/tekton-catlin-lint.yaml

```diff
@@ -80,3 +97,21 @@ spec:
       params:
         - name: gitCloneDepth
           value: $(params.gitCloneDepth)
+    - name: post-comment
+      taskRef:
+        name: github-add-comment
+      when:
+        - input: $(tasks.lint-catalog.results.catlin-result)
+          operator: "notin"
+          values: ["no"]
+      params:
+        - name: COMMENT
+          value: $(tasks.lint-catalog.results.changed-tasks)
+        - name: GITHUB_TOKEN_SECRET_NAME
+          value: bot-token-github
+        - name: GITHUB_TOKEN_SECRET_KEY
+          value: bot-token
```

Comment:
is there a secret setup which contains the `base64` personal access token? If yes, then please help me with the correct values that I can substitute over here :slightly_smiling_face:

Reply:
The values you have in there are already ok